java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/ui/NotifyDescriptorAdapter.java

```diff
@@ -222,6 +222,11 @@ private void mapDescriptorOptions() {
                 text = mapDescriptorOption(o);
             }
             if (text != null) {
+                // Discard ampersands, LSP spec does not support such pattern.
+                int mn = Mnemonics.findMnemonicAmpersand(text);
```

Comment:
There is [cutAmpersand](https://bits.netbeans.org/12.0/javadoc/org-openide-awt/org/openide/awt/Actions.html#cutAmpersand-java.lang.String-) method.

Reply:
`cutAmpersand` is better (handles japanese locale). Thanks ! Fixed in 17ddace